tasks/us/census/tiger.py

```diff
@@ -503,6 +503,9 @@ def run(self):
         session.execute('CREATE INDEX ON {output} USING GIST (the_geom)'.format(
             output=self.output().table))
 
+    def original_task(self):
+        return self.requires()
```

Comment:
Why is this a better option than passing the name through the queries which are related with the geoid? I think that create tables with the needed fields is the regular way to go. 

I know I'm a pain in the ass with this part but we're passing for example aland and awater through all the queries and in order countries we're doing the same so I want to be sure we're doing this "new thing" for a good reason

Reply:
We can propagate through the tables. This just felt easier to do during development, as you only need to alter one query depending on if the dataset has name or not, instead of all intermediate ones. Also, it's a bit faster, since the intermediate tables are smaller (and fit better in cache), but the effect is negligible (a few seconds).

I'm not opposed to changing this to propagate name through the needed parts of the query. However, we still need to propagate whether the task needs a name or not, either via requires (similar to this) or a global configuration dict.